### ISSUE TITLE
Unable to navigate with arrows out of SplitButton in ToolBar overflow dropdown

### ISSUE THREAD

**Issue by PekoPPT** [telerik/kendo-ui-core#5402]
### Bug report

The keyboard navigation of the Toolbar component doesn't work correctly when there are two SplitButtons in the overflow container

### Reproduction of the problem
1. Open [this Dojo project](https://dojo.telerik.com/UvAsayIs). (It has 2 SplitButtons in the Overflow menu)
2. Use the **Tab button** to navigate to the overflow and open in using the ALT + Arrow down keys combination. The Overflow will open and with the keyboard arrows, we can move down it.
3. The keyboard navigation is limited in the first SplitButton and the second SplitButton cannot be accessed via keyboard navigation

The above is a regression introduced with the 2018 R1 version 

### Current behavior
The keyboard navigation is not working correctly when there are two or more SplitButtons in the overflow. 

### Expected/desired behavior
The keyboard navigation should be working correctly for the Toolbar no matter the elements it contains.  
[Here is a Dojo example demonstrating the correct behavior](https://dojo.telerik.com/oWEFutIv). Use the steps described in the "Reproduction" section to see the correct behavior.

### Environment

* **Kendo UI version:** 2019.3.1023
* **jQuery version:** x.y
* **Browser:** [all]

**Comment by veselints** [comment]
Related to: https://github.com/telerik/kendo-ui-core/issues/1479
Make sure the above scenario also behaves properly.

**Comment by Dimitar-Goshev** [comment]
The related scenario works as expected as well.
